package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "svelte-tiptap",
   "description": "Svelte components for tiptap v2",
-  "version": "1.1.3",
+  "version": "2.0.0",
```

Comment:
Please revert this version bump alone, will do it while publishing workflow

Reply:
done